composer.json

```diff
@@ -46,7 +47,7 @@
 	"require": {
 		"php": ">=5.5.0",
 		"composer/installers": "1.*,>=1.0.1",
-		"mediawiki/semantic-media-wiki": "~2.1|~3.0",
+		"mediawiki/semantic-media-wiki": "~2.1",
```

Comment:
If you do this, there may be no stable SRF version available to go with SMW 3.0. At least as long as it takes SRF to catch up, which was several months for 2.5.